arviz/plots/forestplot.py

```diff
@@ -208,7 +208,7 @@ def plot_forest(
         idx += 1
 
     for ax in axes:
-        ax.grid(False)
+        ax.grid(False, axis="x")
```

Comment:
I would suggest to do: 

        if kind == "ridgeplot": 
            ax.grid(False)
        else:
            ax.grid(False, axis="y")

But I understand if we do not want to make a difference between `ridgeplot` and `forestplot`